spec/support/database_cleaner.rb

```diff
@@ -4,11 +4,13 @@
     DatabaseCleaner.clean_with(:truncation)
```

Comment:
cool :)

You may also update database cleaner gem to latest version

Gemfile
```gem 'database_cleaner', git: 'git@github.com:DatabaseCleaner/database_cleaner.git'```

@KirillKayumov